README.md

```diff
@@ -8,71 +8,117 @@
   <em>CLI to work with your AsyncAPI files. Currently supports validation, but it is under development for more features.</em>
 </p>
 
-### Install
-```bash
-$ npm install --global @asyncapi/cli
+<p align="center">
+<a href="https://github.com/asyncapi/cli/blob/master/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/asyncapi/cli"></a>
+<a href="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml">
+<img src="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml/badge.svg" alt="PR testing - if Node project"  />
+</a>
+<a href="https://www.npmjs.com/package/@asyncapi/cli">
+<img alt="npm" src="https://img.shields.io/npm/dw/@asyncapi/cli">
+</a>
+
+</p>
+
+## Table of contents
+
+1. [Getting Started](#getting-started)
```

Comment:
make toc generateable (if such word even exists 😄 ) like we do in other repos, example https://github.com/asyncapi/generator/blob/master/package.json#L27

Reply:
Installing `markdown-toc` I came across 5 vulnerabilities should I audit fix it. 

![image](https://user-images.githubusercontent.com/41781438/132811979-f0385a69-d12c-4b15-9318-19b028575f0c.png)